content/reports/1000_IBM Power - Virtual Machines/011_All PowerHMC VMs.yaml

```
@@ -0,0 +1,86 @@
+---
+  title: IBM PowerHMC VMs
+  rpt_group: Custom
+  rpt_type: Custom
+  priority:
+  db: ManageIQ::Providers::InfraManager::Vm
+  cols:
+  - name
+  - os_image_name
+  - ram_size
+  - entitled_processors
+  - processor_pin_policy
```

```suggestion
  - processor_share_type
```
`processor_pin_policy` isn't used by HMC VMs (so values in this column will always be blank). Instead we could use `processor_share_type` (which is a [virtual column](https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template.rb#L243) that maps to advanced_settings `processor_type` set in the parser [here](https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/blob/master/app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb#L232))